examples/history-api-fallback/file

```diff
@@ -0,0 +1 @@
+file
```

Comment:
Could you make this "file.txt", so the browser doesn't want to download the file if you click it?